plugins/aws/cli_provisioner.go

```diff
@@ -0,0 +1,58 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/1Password/shell-plugins/sdk"
+)
+
+const defaultProfileName = "default"
+
+type awsCLIProvisioner struct {
+}
+
+func NewAwsCLIProvisioner() sdk.Provisioner {
+	return awsCLIProvisioner{}
+}
+
+func (p awsCLIProvisioner) Provision(ctx context.Context, in sdk.ProvisionInput, out *sdk.ProvisionOutput) {
+	profileName := defaultProfileName
+
+	if profile := stripAndReturnProfileFlag(out); profile != nil {
+		profileName = *profile
+	} else if profile := os.Getenv("AWS_PROFILE"); profile != "" {
```

Comment:
I don't think `AWS_PROFILE` is CLI-exclusive, so we should probably move that one down to the STS provisioner.

Reply:
Correct. Done, thanks!